src/views/Hearing/Sidebar.js

```diff
@@ -102,17 +104,29 @@ class Sidebar extends React.Component {
               <div className="sidebar-section contents">
                 <h4><FormattedMessage id="table-of-content"/></h4>
                 <ListGroup>
-                  <ListGroupItem href="#hearing">
+                  {!isQuestionView && <ListGroupItem className={currentlyViewed === '#hearing' && 'active'} href="#hearing">
                     <FormattedMessage id="hearing"/>
                   </ListGroupItem>
-                  {sectionGroups.map((sectionGroup) => (
-                    <ListGroupItem href={"#hearing-sectiongroup-" + sectionGroup.type} key={sectionGroup.type}>
+                  }
+                  {isQuestionView && sectionGroups[0].sections.map((section) => (
+                    <Link to={getSectionURL(hearing.slug, section)}>
+                      <ListGroupItem className={`${section.id}${activeSection.id === section.id ? ' active' : ''}`} href={"#hearing-sectiongroup-" + section.id} key={section.id}>
+                        <div>
+                          {getAttr(section.title, activeLanguage)}
+                          {'     '}
+                          <Icon name="comment-o"/>{section.n_comments}
+                        </div>
+                      </ListGroupItem>
+                    </Link>
+                  ))}
+                  {!isQuestionView && sectionGroups.map((sectionGroup) => (
+                    <ListGroupItem className={currentlyViewed === '#hearing-sectiongroup' && 'active'} href={"#hearing-sectiongroup-part"} key={sectionGroup.type}>
```

Comment:
Here, the section group is highlighted when hovering, but does not act as link when clicked.